admin/static/js/sales_analytics/sa-products-view.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var keen = require('keen-js');
+var ss = require('simple-statistics');
+
+var SalesAnalytics = function() {
+    var self = this;
+
+    self.keenClient = new keen({
+        projectId: keenProjectId,
+        readKey : keenReadKey
+    });
+
+    self.keenFilters = {
+        nullUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: null
+        },
+        inactiveUserFilter: {
+            property_name: 'user.id',
+            operator: 'ne',
+            property_value: ''
+        }
+    };
```

Comment:
There used to be only one file, but I move them to different pages. They are suppose be in {{utils.js}} and exported for other js to require. But I haven't got time to do it yet.